### DISCUSSION TITLE
Tagging/Untagging nodes in bulk

### DISCUSSION THREAD

**Discussion by biswajit-goswami** [freeplane/freeplane#1139] [Help] [answered]
I want to tag a bunch of nodes with some keyword/text so that I can easily filter them out based on that tag. Similarly, I should be able to untag those nodes in bulk. Is this feasible with the current features?
Eg ..lets say I have a keyword 'ResearchTopic1' and I want to tag multiple nodes in bulk...then be able to filter those nodes using that keyword later...and thereafter, remove the tag from 1 or more of the nodes.

**Comment by macmarrum** [comment, marked as answer]
I would use `Edit->Node properties->Find and replace attributes…`. It allows to add an attribute with a particular value (your tag operation), or remove it (your untag operation). It can work on **Selected nodes** or on **All visible nodes**. I use the last one after I filter my mindmap to show only my desired nodes.
